stdlib/ysh/args.ysh

```diff
@@ -185,6 +185,17 @@ func parseArgs(spec, argv) {
             if (_status !== 0) {
               error "Expected Int after '$arg', got '$[argv[i]]'" (code=2)
             }
+          } elif (flag.type is List[Int]) {
+            setvar i += 1
+            if (i >= len(argv)) {
+              error "Expected Int after '$arg'" (code=2)
+            }
+
+            setvar value = get(args, flag.name, [])
+            try { call value->append(int(argv[i])) }
```

Comment:
I kinda want to "compress" this whole `try { ... }; if ...` in each of the 3 blocks, as well as some other things, but since this has good tests, it should be easy to do